t/40blobslarge.t

```diff
@@ -0,0 +1,96 @@
+use strict;
+use warnings;
+
+use Test::More;
+use DBI;
+use vars qw($test_dsn $test_user $test_password);
+use lib '.', 't';
+require 'lib.pl';
+
+sub ShowBlob($) {
+    my ($blob) = @_;
+    my $b;
+    for (my $i = 0;  $i < 8;  $i++) {
+        if (defined($blob)  &&  length($blob) > $i) {
+            $b = substr($blob, $i*32);
+        }
+        else {
+            $b = "";
+        }
+        note sprintf("%08lx %s\n", $i*32, unpack("H64", $b));
+    }
+}
```

Comment:
This sub is never used.

Reply:
Thank you for pointing out. I've removed it and reworded comments little bit